cmd/utils/flags.go

```diff
@@ -234,8 +234,8 @@ var (
 		Name:  "lightkdf",
 		Usage: "Reduce key-derivation RAM & CPU usage at some expense of KDF strength",
 	}
-	WhitelistFlag = cli.StringFlag{
-		Name:  "whitelist",
+	AllowListFlag = cli.StringFlag{
+		Name:  "allowlist",
```

Comment:
This is where the breaking change is.

I would check what is the Geth process for deprecating/replacing a flag and follow the same procedure.

Reply:
Please see the next commit if it is OK like this

Reply:
LGTM